docs/root/configuration/http_filters/lua_filter.rst

```diff
@@ -312,6 +312,57 @@ Returns the current request's underlying :repo:`connection <include/envoy/networ
 
 Returns a :ref:`connection object <config_http_filters_lua_connection_wrapper>`.
 
+importPublicKey()
+^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+  
+  pkey = handle:importPublicKey(keyder, keyderLenght)
+
+Returns public key which is used by :ref:`verifySignature <verify_signature>` to verify digital signature. 
+
+.. attention::
+
+  Remember to call :ref:`releasePublicKey <release_public_key>` when *pkey* is not needed to avoid memory leak.
+
+.. _release_public_key:
+
+releasePublicKey()
+^^^^^^^^^^^^^^^^^^
+
+.. code-block:: lua
+  
+  handle:releasePublicKey(pkey)
+
+Free the resource of *pkey*
+
+decodeBase64()
```

Comment:
Lua is perfectly capable of doing base64 processing on it's own, I don't believe this should be added as an API.